documentation/scope-commands.md

```diff
@@ -4,29 +4,53 @@ Below is a detailed list outlining all of the available scope commands, includin
 
 Other than moving the data block or accepting handoffs, most commands require that the radar target be under your scope's control.
 
-Please note that the bracketed words (and brackets themselves) used in the syntax examples should be replaced with the appropriate data. Also note that `CID` was used in the examples, but aircraft may be referenced with any of ~~(a) the CID~~, (b) the full callsign, or ~~(c) the squawk code~~.
+Please note that the bracketed words (and brackets themselves) used in the syntax examples should be replaced with the appropriate data. Note that in lieu of having to manually type the `callsign` into the command input, it is also possible to click on the aircraft on the radar scope; doing so will automatically append its callsign to the command input and execute the completed command.
+
+While `callsign` is used in the examples, it is envisioned that in future it will be possible for  aircraft to also be referenced by their CID or squawk code.
```

Comment:
- Double space between `for` and `aircraft`.
- it currently works with a click, the full callsign, or the squawk code. Only the CID is not yet implemented for this.

Reply:
fixed